acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/jsonrpc/Ibft.java

```diff
@@ -12,19 +12,84 @@
  */
 package tech.pegasys.pantheon.tests.acceptance.dsl.jsonrpc;
 
+import tech.pegasys.pantheon.ethereum.core.Address;
+import tech.pegasys.pantheon.tests.acceptance.dsl.condition.Condition;
+import tech.pegasys.pantheon.tests.acceptance.dsl.condition.ibft.ExpectProposals;
+import tech.pegasys.pantheon.tests.acceptance.dsl.condition.ibft.ExpectValidators;
 import tech.pegasys.pantheon.tests.acceptance.dsl.node.PantheonNode;
+import tech.pegasys.pantheon.tests.acceptance.dsl.transaction.eth.EthTransactions;
+import tech.pegasys.pantheon.tests.acceptance.dsl.transaction.ibft.IbftTransactions;
 
 import java.util.Arrays;
 import java.util.Comparator;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.stream.Collectors;
+
+import com.google.common.collect.ImmutableMap;
 
 public class Ibft {
 
+  private final EthTransactions eth;
+  private final IbftTransactions ibft;
+
+  public Ibft(final EthTransactions eth, final IbftTransactions ibft) {
+    this.eth = eth;
+    this.ibft = ibft;
+  }
+
   public List<PantheonNode> validators(final PantheonNode[] nodes) {
     final Comparator<PantheonNode> compareByAddress =
         Comparator.comparing(PantheonNode::getAddress);
     List<PantheonNode> pantheonNodes = Arrays.asList(nodes);
     pantheonNodes.sort(compareByAddress);
     return pantheonNodes;
   }
+
+  public ExpectValidators validatorsEqual(final PantheonNode... validators) {
+    return new ExpectValidators(ibft, validatorAddresses(validators));
+  }
+
+  private Address[] validatorAddresses(final PantheonNode[] validators) {
+    return Arrays.stream(validators).map(PantheonNode::getAddress).sorted().toArray(Address[]::new);
+  }
+
+  public Condition noProposals() {
+    return new ExpectProposals(ibft, ImmutableMap.of());
+  }
+
+  public ProposalsConfig proposalsEqual() {
```

Comment:
Ibft is using pending votes instead of proposals for this concept, so I think it would be clearer in the ibft tests if we used the term pending votes

Reply:
done.